### PR TITLE
[SDK 450] Internal event records with record event

### PR DESCRIPTION
**Pull request by ZahidZafar**

### CODE REVIEW

**Comment by ZahidZafar** [comment]
> This does not implement the current "internal event" spec. There should be checks and support for all countly internal events

Views, StarRating, Push, Feedback(Survey, Orientation), and Clicks(action event) use internal event to record respective event.
Windows SDK does have only 'Views' feature, reset of them are not even declared in Consent features list.
Push and StarRating wouldn't be part of windows SDK I guess.
We have to add remain features in Consent enum to work through internal events.

File: countlyCommon/countlyCommon/CountlyBase.cs [hunk 1]
```diff
@@ -1024,8 +1036,11 @@ public async Task ChangeDeviceId(String newDeviceId, bool serverSideMerge = fals
         internal bool consentRequired = false;
         internal Dictionary<ConsentFeatures, bool> givenConsent = new Dictionary<ConsentFeatures, bool>();
 
-        public enum ConsentFeatures { Sessions, Events, Location, Crashes, Users, Views };
-
+        public enum ConsentFeatures
+        {
+            Sessions, Events, Location, Crashes, Users, Views, Push, Feedback,
+            StarRating, RemoteConfig
```
Comment (ArtursKadikis) [inline countlyCommon/countlyCommon/CountlyBase.cs:1042]: this should all be on a single line

**Review by ArtursKadikis — CHANGES_REQUESTED** [review]
Perform code styling

File: countlyCommon/countlyCommon/CountlyBase.cs (hunk 1 above)
Comment (ZahidZafar) [reply countlyCommon/countlyCommon/CountlyBase.cs:1042]: I performed formating, it is also acceptable.

**Pull request merged**